srcpkgs/GoLand/template

```diff
@@ -0,0 +1,55 @@
+# Template file for 'GoLand'
+pkgname=GoLand
+version=2019.1.1
+revision=1
+wrksrc="GoLand-${version}"
```

Comment:
This is default, remove.

Reply:
Assuming you mean `wrksrc="GoLand-${version}"` here and not any of the other highlighted variables.

Reply:
Yes, I meant default of wrksrc is pkgname-version.